apps/frontend-demo/.env.local.example

```
@@ -1,6 +1,6 @@
-RESEND_API_KEY=re_xxxxxxxxxx
-CONTACT_EMAIL=your-email@example.com
-
-# From email - must be from a verified domain in Resend
-# For testing you can use onboarding@resend.dev
-RESEND_FROM_EMAIL=onboarding@resend.dev
\ No newline at end of file
+NEXT_PUBLIC_MEDUSA_BACKEND_URL=http://localhost:9000
+NEXT_PUBLIC_MEDUSA_PUBLISHABLE_KEY=pk_f2ac1c3efaf10d8a861f0623a9d6e7697fad3c2b239b174ddca61ebff7c9005b
+# Prefer Czech region by default (falls back to CZK/EUR/first region if missing)
+NEXT_PUBLIC_DEFAULT_REGION_ID=reg_01KAGV84E85P9CX48N7YNMR8NF
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NEXT_PUBLIC_DEFAULT_REGION_ID key should go before the NEXT_PUBLIC_MEDUSA_BACKEND_URL key

(UnorderedKey)